spec/factories/issues.rb

```diff
@@ -4,5 +4,11 @@
     author { "factory_bot" }
     category { Category.issue }
     node { Project.new.issue_library }
+
+    factory :tagged_issue do
```

Comment:
why `factory` and not `trait`?

Reply:
No actual reason, it's the first thing that came to mind. I do agree that trait is better here for the following reasons: 1) it can be mixed and matched with future traits we want to implement and 2) semantics.